pkg/diagnostics/network/objects.go

```diff
@@ -13,21 +13,23 @@ import (
 )
 
 const (
-	busyboxImage               = "docker.io/busybox"
+	diagnosticsImage           = "openshift/diagnostics-deployer"
```

Comment:
I don't think this is affected by imageConfig format. 
Do you prefer config option for the diagnostics image? and then you can pass 'openshift3/ose-diagnostics-deployer:latest'